src/main/scala/akka_debugging/database/DatabaseUtils.scala

```diff
@@ -1,26 +1,16 @@
 package akka_debugging.database
 
-import java.util.UUID
-
 import com.typesafe.config._
 import org.slf4j.{Logger, LoggerFactory}
 
 import scala.concurrent.duration._
 import scala.concurrent.{Await, Future}
 
-case class CollectorDBMessage(uuid: UUID,
-                              actor: String,
-                              message: String,
-                              stackTrace: String,
-                              id: Option[Long] = None)
-
-
-case class CollectorDBExceptionMessage(uuid: UUID,
-                                       actor: String,
-                                       exception: String,
-                                       stackTrace: String,
-                                       id: Option[Long] = None)
+case class CollectorDBMessage(id: Int,
```

Comment:
For now it's only simplified version. I think we can leave it for now and change it in appropriate way in the future.